sources/xfscli/src/error.rs

```diff
@@ -0,0 +1,55 @@
+use snafu::Snafu;
+
+pub type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Debug, Snafu)]
+#[snafu(visibility(pub))]
+#[repr(i32)]
+pub enum Error {
+    #[snafu(display("xfs fsck errror: xfs_repair was not found!"))]
+    CommandUnavailable,
+
+    #[snafu(display("{} does not exist", target))]
+    DeviceNotFound { target: String },
+
+    #[snafu(display(
+        "xfs fsck errror: The filesystem log is dirty, mount it to recover \
+                the log. If that fails, refer to the section DIRTY LOGS in the \
+                xfs_repair manual page."
```

Comment:
This doesn't do what you think it does, I think the `\` is going to be included in the long string. Can you confirm the correct output shows up for this error?